service/debian/control

```diff
@@ -4,6 +4,7 @@ Priority: optional
 Maintainer: Christopher M. Cantalupo <christopher.m.cantalupo@intel.com>
 Build-Depends: debhelper-compat (=13),
                build-essential,
+               datacenter-gpu-manager,
```

Comment:
Does this also need to be in `Depends`? I think this package is combined runtime and build stuff.

Reply:
Separated the PR into documentation (the current focus) vs. implementation